README.md

```diff
@@ -3,3 +3,34 @@
 ## Project Overview
 
 This C++ project, developed for the Functional Programming course (FPROG), utilizes functional programming concepts to analyze literary texts. The program reads a large text file (e.g., "War and Peace" by Leo Tolstoy) and categorizes its chapters as either war-related or peace-related based on the density of specific terms. This is achieved using two additional text files: one containing war-related terms and another with peace-related terms.
+
+
+## How to compile
+
+We are using cmake to compile the project. To compile the project, run the following commands in the project's root directory:
+
+``` 
+mkdir build
+cd build
+cmake ..
+make
+```
+
+## How to run
+
+To run the program, run the following command in the build directory:
+
+```
+./build/war_and_peace <peace_terms_file> <war_terms_file> <war_and_peace_file>
```

Comment:
damit du den command auch in der Übersicht gleich siehst hier nochmal ein Kommentar 🐌